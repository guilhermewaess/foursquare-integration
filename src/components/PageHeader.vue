
<template>
  <v-jumbotron dark
               src="https://s3.eu-central-1.amazonaws.com/euobs-media/107425944ee4ad8015c1082c05c7df8a.jpg">
    <v-container fill-height>
      <v-layout align-center>
        <v-flex text-xs-center>
          <h3 class="display-3">Welcome to Foursquare Venue</h3>
          <h1 class="body-1">
            <!-- eslint-disable-next-line max-len -->
            **Please, allow chrome to access you location, otherwise we will consider you in the center of Amsterdam**
          </h1>
        </v-flex>
      </v-layout>
    </v-container>
  </v-jumbotron>
</template>

<script>
export default {
  name: 'PageHeader',
};
</script>

