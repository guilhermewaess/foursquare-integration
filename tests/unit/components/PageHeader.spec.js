import { shallowMount } from '@vue/test-utils';
import PageHeader from '@/components/PageHeader.vue';

describe('PageHeader', () => {
  let component;
  beforeEach(() => {
    component = shallowMount(PageHeader, {
      stubs: ['v-jumbotron', 'v-container', 'v-layout', 'v-flex'],
    });
  });
  it('should have name', () => {
    expect(component.name()).toEqual('PageHeader');
  });
  describe('snapshot', () => {
    it('should render', () => {
      expect(component.element).toMatchSnapshot();
    });
  });
});
