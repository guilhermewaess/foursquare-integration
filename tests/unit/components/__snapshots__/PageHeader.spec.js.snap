// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader snapshot should render 1`] = `
<v-jumbotron-stub
  dark=""
  src="https://s3.eu-central-1.amazonaws.com/euobs-media/107425944ee4ad8015c1082c05c7df8a.jpg"
>
  <v-container-stub
    fill-height=""
  >
    <v-layout-stub
      align-center=""
    >
      <v-flex-stub
        text-xs-center=""
      >
        <h3
          class="display-3"
        >
          Welcome to Foursquare Venue
        </h3>
         
        <h1
          class="body-1"
        >
          
          **Please, allow chrome to access you location, otherwise we will consider you in the center of Amsterdam**
        
        </h1>
      </v-flex-stub>
    </v-layout-stub>
  </v-container-stub>
</v-jumbotron-stub>
`;
